C-Plus-Plus/math/n_queen.cpp

```diff
@@ -0,0 +1,100 @@
+//This problem is of placing n queens on n*n chessboard which is solved using backtracking so no two queens ca attack each other
```

Comment:
It should be `can` not `ca`. Add more description

Reply:
made the changes @HarshCasper pls check